### DISCUSSION TITLE
Environment Variables required for MongoDB

### DISCUSSION THREAD

**Discussion by adityaxpique** [Atri-Apps/traffic-history#4] [General]
You need to provide three environment variables for proper working of the program. The three environment variables are:-
1. USER - Pass the github token in the following format USER="Bearer {token}"
2. DATABASE - Pass the name of the database you want to use, in this case it will be MongoDB.
3. DB_URI - Pass the URI needed to connect to the database.

For MongoDB, the DB URI looks like this -
mongodb+srv://\<username\>:\<password\>@cluster0.d7lrzyw.mongodb.net/?retryWrites=true&w=majority

**Comment by cruxcode** [comment]
@adityaxpique do we need both `DATABASE` and `DB_URI`? I thought `DB_URI` contains the database name at its end.
